bot/zephyr.py

```diff
@@ -158,38 +162,25 @@ def apply_overlay(zephyr_wd, base_conf, cfg_name, overlay):
     os.chdir(cwd)
 
 
-autopts2board = {
-    None: None,
-    'nrf52': 'nrf52840dk_nrf52840',
-    'reel_board' : 'reel_board'
-}
-
-
-def get_tty_path(name):
-    """Returns tty path (eg. /dev/ttyUSB0) of serial device with specified name
+def get_tty_path(name, board, board_id):
+    """Returns symlink by-id tty path (eg. /dev/ttyUSB0) of serial device with specified name
```

Comment:
'with specified name'? We don't use its name to retrieve the path any longer.
Suggestion: You can think of removing the name from the parameters.

Reply:
done 👍